AdyenComponents/Affirm/AffirmComponent.swift

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+/// A component that provides a form for Affirm payment.
+public final class AffirmComponent: AbstractPersonalInformationComponent, Observer {
+    
+    // MARK: - Items
+    
+    /// :nodoc:
+    private let personalDetailsHeaderItem: FormLabelItem
+    
+    /// :nodoc:
+    public let deliveryAddressToggleItem: FormToggleItem
+    
+    /// :nodoc:
+    public let deliveryAddressItem: FormAddressItem
+    
+    // MARK: - Properties
+    
+    private var isDeliveryAddressEnabled: Bool {
```

Comment:
you only use this once, is it really needed? 🤔

Reply:
Removed to use toggle value direcly